systest/1million/docker-compose.yml

```diff
@@ -1,64 +1,5 @@
-# Auto-generated with: [./compose -a 3 -z 1 -r 1 --port_offset=0 --expose_ports=false --alpha_volume data:/data --zero_volume data:/data -w --mem= --names=false -O ../systest/1million/docker-compose.yml]
-#
```

Comment:
No need to split this file into 2 files. You can pass the specific containers you want to spawn.

Reply:
How would t.go know about this? Like which containers to start or how many zeros/alphas are going to be there? I believe if we can do it, it will be bit tricky. Let's communicate offline and see if we can manage this better.